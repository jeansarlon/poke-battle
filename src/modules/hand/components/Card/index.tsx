import React from 'react';
import { ICard } from 'pokemon-tcg-sdk-typescript/dist/sdk';

const Card = ({ name }: ICard) => (
  <div className="card">
    {
      name
        ? (<span className="card-name">{name}</span>)
        : (<span>nao hรก card</span>)

    }
  </div>
);
export default Card;
