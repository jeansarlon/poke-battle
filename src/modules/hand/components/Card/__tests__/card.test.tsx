import React from 'react';
import { render } from '@testing-library/react';
import Card from '..';

describe('Card', () => {
  test('should mount Component', () => {
    const { container } = render(<Card />);

    const element = container.querySelector('.card');
    expect(element).toBeDefined();
    expect(container).toMatchSnapshot();
  });
});

