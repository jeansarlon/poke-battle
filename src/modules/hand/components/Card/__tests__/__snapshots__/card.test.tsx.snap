// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should mount Component 1`] = `
<div>
  <div
    class="card"
  >
    cards
  </div>
</div>
`;
